ActionScript/ActionScript.sublime-build

```diff
@@ -1,12 +1,16 @@
 {
-	"selector": "source.actionscript",
-	"cmd": [
-		"mxmlc", 
-		"${file}",
-		"-library-path+=${project_path}/libs",
-		"-output", "${project_path}/bin/${project_base_name}.swf",
-		"-debug=false",
-		"-static-link-runtime-shared-libraries=true"
-	],
-	"file_regex": "^(.+?)\\(([0-9]+)\\): col: (([0-9]+))(.*)$"
-}
\ No newline at end of file
+    "selector": "source.actionscript",
+    "file_patterns": [
+        "*.as"
+    ],
+    "cmd": [
```

Comment:
Most other build systems start with the command to run. Maybe this one should do that as well?